packages/elements/src/components/ui/action-menu/ActionMenu.tsx

```diff
@@ -1,30 +1,36 @@
 import { BoxProps, Column } from "@stenajs-webui/core";
 import * as React from "react";
 import { ReactNode } from "react";
-import { ActionMenuTheme, defaultActionMenuTheme } from "./ActionMenuTheme";
+import styles from "./ActionMenu.module.css";
 import { FocusScope } from "@react-aria/focus";
+import cx from "classnames";
+
+export type ActionMenuVariant = "standard" | "outlined";
 
 export interface ActionMenuProps {
   top?: ReactNode;
-  theme?: ActionMenuTheme;
-  width?: string;
+  width?: BoxProps["width"];
+  variant?: ActionMenuVariant;
   shadow?: BoxProps["shadow"];
   trapFocus?: boolean;
 }
 
 export const ActionMenu: React.FC<ActionMenuProps> = ({
-  theme = defaultActionMenuTheme,
   top,
   children,
-  width = "200px",
+  width,
   shadow,
+  variant = "standard",
   trapFocus,
 }) => {
+  if (!children) {
+    return null;
+  }
+
   return (
     <FocusScope contain={trapFocus}>
       <Column
-        background={theme.menuBackground}
-        borderRadius={theme.borderRadius}
+        className={cx(styles.ActionMenu, styles["menu-" + variant])}
```

Comment:
In other components, we have had the variant name as class name, not prefixed.
Not very important, but I thought I'd mention it for the sake of consistency :)